### ISSUE TITLE
There is a problem with the way the most recently uploaded image is downloaded first.

### ISSUE THREAD

**Issue by Daorf** [RipMeApp/ripme#1893]
* Ripme version: V1.7.95
* Java version: 
* Operating system: 

* Exact URL you were trying to rip when the problem occurred:
* Please include any additional information about how to reproduce the problem:

Ripme uses an ascending method where the first image downloaded is numbered first and the last image downloaded is numbered last.
And can't you change the image download order so that the first image is downloaded first and the most recent image is downloaded last?
The current Ripme is a method in which the most recently uploaded image is downloaded first, but in this case, there is a problem when trying to download a newly uploaded image.

**Comment by Katsufaux** [comment]
I believe this can be changed by toggling "Preserve order", but I could be wrong.

**Comment by rautamiekka** [comment]
No, the 'Preserve order' only includes the website-specific upload order ID in the filename, it doesn't affect the order of downloading.

**Comment by Daorf** [comment]
> _So, is Ripme's Reverse Order not working properly?_
